unifi/Dockerfile

```diff
@@ -14,10 +14,10 @@ RUN \
         libcap2=1:2.32-1 \
         logrotate=3.14.0-4ubuntu3 \
         mongodb-server=1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 \
-        openjdk-11-jdk-headless=11.0.19+7~us1-0ubuntu1~20.04.1 \
+        openjdk-17-jdk-headless=17* \
```

Comment:
This needs to be pinned strictly.

Reply:
Ok this package doesn't even exist..

Reply:
Ok, the amd64 version has some security updates.

Unless we add some logic here to switch based on architecture... it will be ugly.

Let's slide this in as is.